rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -1211,6 +1211,36 @@ void save_saveLastRetiredFederationP2SHScript_afterRSKIP186_getsValueFromStorage
 
     }
 
+    @Test
+    void save_saveFederationElection_shouldSaveInStorage() {
+
+        // Arrange
+
+        AddressBasedAuthorizer authorizer = getTestingAddressBasedAuthorizer();
+        StorageAccessor storageAccessor = new InMemoryStorage();
+        FederationStorageProvider federationStorageProvider = new FederationStorageProviderImpl(storageAccessor);
+
+        // Act
+
+        ABICallElection defaultFederationElection = federationStorageProvider.getFederationElection(authorizer);
+        byte[] defaultFederationElectionSerialized = BridgeSerializationUtils.serializeElection(defaultFederationElection);
+        ABICallSpec abiCallSpec = new ABICallSpec("function1", new byte[][]{});
+        RskAddress voterAddress = new RskAddress("9be6f6735c4d59c10240d4987414fb686c6b7323");
```

Comment:
shouldn't these be part of the _Arrange_ section?

Reply:
Updated.